install/helm/agones/templates/extensions.yaml

```diff
@@ -16,12 +16,44 @@
 {{- $altName1 := printf "agones-controller-service.%s"  .Release.Namespace }}
 {{- $altName2 := printf "agones-controller-service.%s.svc" .Release.Namespace }}
 {{- $cert := genSignedCert $cn nil (list $altName1 $altName2) 3650 $ca }}
+---
+{{- if .Values.agones.registerApiService }}

```

Comment:
is there a reason you don't want that?

also perhaps "registerAllocationApiService" ?

Reply:
GCP Marketplace usually requires a different install step, so there are switches to turn various pieces off if people need to break the glass and copy things into their own install process.

@bbf to confirm.

Reply:
Regarding the rename comment - we have `registerWebhooks` for all our webhooks - I don't think we need to be that granular. If there ever end up being more APIServices, they could also go here.